frontend/src/pages/homepage/components/truck-list/truck-list.tsx

```diff
@@ -0,0 +1,58 @@
+import { Spinner } from '~/libs/components/components.js';
+import { TowTruckCard } from '~/libs/components/tow-truck-card/tow-truck-card.js';
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import { useAppSelector, useMemo } from '~/libs/hooks/hooks.js';
+import { selectTrucks } from '~/slices/trucks/selectors.js';
+
+import styles from './styles.module.scss';
+
+type MockDataRating = {
+  averageRating: number;
+  reviewCount: number;
+};
+
+const getMockRating = (): MockDataRating => {
+  const averageRating = Math.floor(Math.random() * 3) + 3;
+  const reviewCount = Math.floor(Math.random() * 15) + 1;
+
+  return {
+    averageRating,
+    reviewCount,
+  };
+};
+
+const getMockDistance = (): number => Math.floor(Math.random() * 50) + 1;
+
+const TruckList: React.FC = () => {
+  const trucks = useAppSelector(selectTrucks);
+
+  const mockData = useMemo(() => {
+    const data = [];
+    // eslint-disable-next-line @typescript-eslint/prefer-for-of
+    for (let index = 0; index < trucks.length; index++) {
+      data.push({ rating: getMockRating(), distance: getMockDistance() });
+    }
+
+    return data;
+  }, [trucks.length]);
+
+  return (
+    <>
+      <h2 className={getValidClassNames('textMd', styles.title)}>Tow Trucks</h2>
+      {trucks.length === 0 && <Spinner />}
```

Comment:
```suggestion
      {!trucks.length && <Spinner />}
```

Reply:
I wanted to do that, but eslint gives an error.

![image (3)](https://github.com/BinaryStudioAcademy/bsa-2023-towhub/assets/91832203/3123680b-9c7b-4c47-84b8-c01540312ba4)

Reply:
Got it 🥲